massa-models/src/test_exports/tools.rs

```diff
@@ -15,7 +15,7 @@ pub fn get_next_slot_instant(
     let mut slot = get_closest_slot_to_timestamp(thread_count, t0, genesis_timestamp, now);
 
     // ignore genesis
-    if slot.period == 0 {
+    if slot.period <= 0 {
```

Comment:
less than 0?

Reply:
This was from the time I wanted to use last_start_period here, but I'm not, so I'll reuse the `==` :)